src/components/record-report-page/record-report-page-pure.tsx

```diff
@@ -4,9 +4,11 @@ import { RouteComponentProps } from 'react-router-dom';
 import SC from './styled';
 import Headline from '../headline';
 import { RecordItem } from './record-item';
-import { Record } from '../../types';
+import { Record, RepresentativesInterface } from '../../types';
 import { fetchAllRecordsRequested } from '../record-list-page/redux/actions';
 import { localization } from '../../lib/localization';
+import { fetchAllRepresentativesRequested } from '../representatives/redux/actions';
```

Comment:
Det jeg egentlig tenkte er at i stedet for referere til redux i en helt annen komponent, lager vi en felles en, som er koblet til redux. Da kan vi wrappe denne komponenten med withRepresentatives hook